scripts/mac/jamf/v2/zluri-mac-install-update-kamf.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# Supported Versions: 3.3.0 and above 
+
+#Config
+ORG_TOKEN="<ORG_TOKEN>"     # Replace <ORG_TOKEN> with organization token
+INTERVAL=600000             
+SCREEN_RECORD=off           # Turn this on if you want to see the option to enable screen record
+SILENT_AUTH=on              # Turn this off if agent should up if authentication is not successful
+LOCAL_SERVER=on             # This is required when browser agent can authenticate with loggedin info from desktop agents
+EXPECTED_VERSION="3.6.3"    # Replace with the latest version
+ZLURI_PKG_URL="https://zluri-prod-agent-builds.s3.us-west-2.amazonaws.com/zluri-3.6.3.pkg"
+
+################################################## DO NOT MAKE ANY MODIFICATION BELOW ##################################################
+
+#Paths
+ZLURI_APP="/Applications/zluri.app"                 # Path of the Zluri app
+ZLURI_DIR="$HOME/Library/Application Support/zluri" # Path to contents of the Zluri app
+CONFIG_FILE="$ZLURI_DIR/client-config.json"         # Path to add client-config file
+TMP_CONFIG_DIR="/tmp/zluritemp"                     # Path for tmp config
+TMP_CONFIG_FILE="/tmp/zluritemp/client-config.json" # Path to add client-config file in tmp
+TMP_ZLURI_FILE="/tmp/zluri-agent.pkg"               # Temp path to add client-config file
+LOG_FILE="/tmp/mdm_script_log.txt"                  # Temp log file
```

Comment:
Suggestion: Instead of storing these log files in tmp folder should we store in some other folder in case we need this to debug later ? 
Thoughts?

Reply:
We will get logs from MDM only. These are just temperory debug logs inside machine which gets removed on restart of machine.